src/plugins/data/public/index.ts

```diff
@@ -303,7 +306,7 @@ export {
   TimeHistoryContract,
 } from './query';
 export * from './ui';
-
+export { FieldFormatsStart } from './field_formats';
```

Comment:
No need to export this I think

Use DataPluginPublicStart['fieldFormats'] where needed